distribution/src/root/config/log4j2.properties

```diff
@@ -22,8 +22,8 @@ appender.console.layout.pattern=${env:LOGGING_PATTERN}
 
 appender.rolling.type=RollingFile
 appender.rolling.name=RollingFile
-appender.rolling.fileName=${sys:jet.home}/logs/hazelcast-jet.log
-appender.rolling.filePattern=${sys:jet.home}/logs/hazelcast-jet.log.%d{yyyy-MM-dd}
+appender.rolling.fileName=logs/hazelcast.log
+appender.rolling.filePattern=logs/hazelcast.log.%d{yyyy-MM-dd}
```

Comment:
Is the lack of the variable intentional?

Reply:
Yes, intentional. We don't have any such variable in IMDG or Hazelcast I believe.

Reply:
Well, we don't have it because [we remove it in this PR](https://github.com/hazelcast/hazelcast/pull/18999/files#diff-d3df82c28b5cc4e00c9596243e36edc85300b6632bb27d0a02d429f3f3b1806eL28) :smile: Perhaps we should keep that argument with renaming to `hazelcast.home`? Otherwise, I believe the files will be looked up in the current directory.

Reply:
If we do it, we need to add it back, we have to add it to [`hz-start.bat`](https://github.com/hazelcast/hazelcast/pull/18999/files#diff-546624a9e308863088569a90126fa245d05b7940f2ce05a6a14a1130a20c034eR44) as well.

Reply:
Jet had a `HazelcastProperty` for it: https://github.com/hazelcast/hazelcast/pull/18999/files#diff-4948e183605089a9e09b8f9ffd3fc40c6664a7bb277f6c53c82d1db43b573db2L68-L76

Reply:
Yes, we decided to remove it, since it was used to auto-enable hot restart, which we said we will not do in HZ - https://hazelcast.slack.com/archives/C01JU7ZJYGP/p1617879751219300

Reply:
Here is where Ufuk suggested we remove it as well - https://hazelcast.slack.com/archives/C01JU7ZJYGP/p1625655877168400?thread_ts=1625652478.160400&cid=C01JU7ZJYGP

Reply:
I do get your point that we might introduce the property `hz.home`. I guess it's only purpose (given that we gave up on the auto-enabling HR use case) would be that it would be used to define the log directory for someone starting with the script.

I'm not sure if that use case really warrants a `HazelcastProperty` or if we can simply allow the user to provide it as an argument to the script.

Reply:
Ok, after some deep thinking, I'll bring it back as `hz.home` for the purpose of the log configuration :D

Reply:
Aaaaand here it is :)

https://github.com/hazelcast/hazelcast/pull/18999/commits/e4db434a8fe36637e35e3fb1384645df23ce8939

Reply:
:man_dancing: